sql/session.proto

```diff
@@ -35,4 +35,7 @@ message Session {
   // Indicates that the above transaction is mutating keys in the
   // SystemDB span.
   optional bool mutates_system_db = 4 [(gogoproto.nullable) = false, (gogoproto.customname) = "MutatesSystemDB"];
+  // Timestamp to be used by SQL in the above transaction. Note: this is not the
+  // transaction timestamp in proto.Transaction above.
+  optional proto.Timestamp unix_timestamp = 5 [(gogoproto.nullable) = false];
```

Comment:
this is not the right type. We should use the driver's `Timestamp` here, not `proto`'s/